src/Dapr.AzureFunctions.Extension/Triggers/DaprTopicTriggerBindingProvider.cs

```diff
@@ -37,36 +37,54 @@ public DaprTopicTriggerBindingProvider(DaprServiceListener serviceListener, INam
                 return Utils.NullTriggerBindingTask;
             }
 
+            // Resolve names in pub/sub, topic, and route from settings
+            if (!this.nameResolver.TryResolveWholeString(attribute.PubSubName, out var pubSubName))
+            {
+                pubSubName = attribute.PubSubName;
+            }
+
             string topic = TriggerHelper.ResolveTriggerName(parameter, this.nameResolver, attribute.Topic);
 
-            // Verison 0.8 only support route to be the same as the topic name
+            if (attribute.Route is null || !this.nameResolver.TryResolveWholeString(attribute.Route, out var route))
+            {
+                route = attribute.Route ?? topic;
```

Comment:
Shouldn't this be route = topic instead of the is null check?

Reply:
I'm not 100% sure I understand what you're asking, but I'll try to answer.

In 0.8 there was a dapr runtime limitation that the topic and route had to be the same. This is no longer the case, but the functions extension was not updated to support it.  I added that support.